pkg/tool/helm_test.go

```diff
@@ -1,65 +1,266 @@
 package tool
```

Comment:
Yes I've considered to restructure with the `pkg/helm/actions` in mind. Though we need to think more carefully about the structure of the program. For example, can we merge `pkg/helm/actions` and `tool` package altogether? Is the current executor package really needed? I deliberately left out the restructure part out of this PR because the lines of change will grow and grow, and makes code reviews harder. But I'm willing to make any further restructuring changes in a separate PR.

Reply:
The changes done so far has exact structure as before, except the function headers (additional parameters etc).

Reply:
Happy to restructure later as long as we do, the distinction was:
 
pkg/helm/actions/ has code which uses the helm SDK.
pkg/tool/ - had code which used helm and oc clients.

so now we have two packages which use SDK's and none use clients. Personally prefer the former but then it was me that created it :-).